src/components/Routes/Find/SavedJobs/SavedJobs.js

```diff
@@ -17,6 +17,32 @@ class SavedJobs extends Component {
     }
   }
 
+  count = 0;
+  componentDidUpdate = (prevProps) => {
+    let jobsLengths = true;
+    let jobsNotEqual = true;
+    let applicationsLengths = true;
+    let applicationsNotEqual = true;
+    if(this.count > 0) {
+      jobsLengths = prevProps.savedJobs.length === 0 && this.props.savedJobs.length === 0;
+      jobsNotEqual = prevProps.savedJobs === this.props.savedJobs;
+      applicationsLengths = prevProps.savedApplications.length === 0 && this.props.savedApplications.length === 0;
+      applicationsNotEqual = prevProps.savedApplications === this.props.savedApplications;
```

Comment:
What you're looking for is something like [_.isEqual](https://lodash.com/docs/4.17.4#isEqual). 

[Here's more to read up on](https://medium.com/@sankalp.lakhina91/understanding-object-equality-in-javascript-ec3e7a2b5b1d)...
> Primitives like strings and numbers are compared by their value, while objects like arrays, dates, and plain objects are compared by their reference.

Reply:
There's also a similar check in `Applications.js` line 30.